packages/terra-form-select/src/_Option.module.scss

```diff
@@ -44,27 +44,27 @@
     background-size: contain;
     display: inline-block; // IE 10
     flex-shrink: 0;
-    height: var(--terra-select-option-icon-height, 1.143rem);
-    margin-right: var(--terra-select-option-icon-margin-right, 0.714rem);
+    height: var(--terra-form-select-option-icon-height, 1.143rem);
+    margin-right: var(--terra-form-select-option-icon-margin-right, 0.714rem);
     visibility: hidden;
-    width: var(--terra-select-option-icon-width, 1.143rem);
+    width: var(--terra-form-select-option-icon-width, 1.143rem);
   }
 
   .is-checkable > .icon {
-    background: var(--terra-select-option-icon-background, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><rect width="48" height="48" rx="6" ry="6" fill="#1c1f21" opacity="0.12"/><path d="M20,36.4,6.7,23.1l3.6-3.6L20,29.4,37.7,11.5l3.6,3.6Z" fill="#0087d1"/></svg>'));
+    background: var(--terra-form-select-option-icon-background, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><rect width="48" height="48" rx="6" ry="6" fill="#1c1f21" opacity="0.12"/><path d="M20,36.4,6.7,23.1l3.6-3.6L20,29.4,37.7,11.5l3.6,3.6Z" fill="#0087d1"/></svg>'));
   }
 
   .is-selected.is-active > .icon {
-    background: var(--terra-select-option-icon-active-background, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><rect width="48" height="48" rx="6" ry="6" fill="#a3dcfd" opacity="0.5"/><path d="M20,36.4,6.7,23.1l3.6-3.6L20,29.4,37.7,11.5l3.6,3.6Z" fill="#0079be"/></svg>'));
+    background: var(--terra-form-select-option-icon-active-background, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><rect width="48" height="48" rx="6" ry="6" fill="#a3dcfd" opacity="0.5"/><path d="M20,36.4,6.7,23.1l3.6-3.6L20,29.4,37.7,11.5l3.6,3.6Z" fill="#0079be"/></svg>'));
   }
 
   .is-checkable:active > .icon {
-    background: var(--terra-select-option-icon-pressed-background, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><rect width="48" height="48" rx="6" ry="6" fill="#a3dcfd" opacity="0.8"/><path d="M20,36.4,6.7,23.1l3.6-3.6L20,29.4,37.7,11.5l3.6,3.6Z" fill="#0079be"/></svg>'));
+    background: var(--terra-form-select-option-icon-pressed-active-background, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><rect width="48" height="48" rx="6" ry="6" fill="#a3dcfd" opacity="0.8"/><path d="M20,36.4,6.7,23.1l3.6-3.6L20,29.4,37.7,11.5l3.6,3.6Z" fill="#0079be"/></svg>'));
```

Comment:
nit: If you drop `pressed` from the other variable names, I'd rename this to `--terra-form-select-option-checkable-active-icon-background` if the linter allows it. I'm not too concerned about this one.

Reply:
https://github.com/cerner/terra-core/pull/2125/commits/08c864fe81968bcf218d28099650960c2f45b51d#diff-f25f4a53a03fa6ff98e47bf17de1a8a9R62